### ISSUE TITLE
Grouped buttons - automatically add aria-pressed attribute

### ISSUE THREAD

**Issue by C-Lodder** [toiletKit/custom-elements#44]
The `aria-pressed` attribute should be automatically added when constructing the custom element, as opposed to manually hard coding it.

https://toiletkit.github.io/custom-elements/#/group-buttons?id=group-buttons-wip

@dgt41 ?
